packages/conform-validitystate/index.ts

```diff
@@ -481,10 +482,10 @@ export function parse<
 	const payloadMap = new Map<keyof Constraints, string | string[]>();
 	const controlMap = new Map<keyof Constraints, Input>();
 	const errorMap = new Map<keyof Constraints, FormError>();
-	const valueMap = new Map<keyof Constraints, any>();
-	// @ts-expect-error FIXME: handle default error type
-	const format: FormatErrorFunction<Constraints, FormError> =
-		schema.formatError ?? defaultFormatError;
+	const valueMap = new Map<keyof Constraints, unknown>();
+	const format =
+		schema.formatError ??
+		(defaultFormatError as FormatErrorFunction<Constraints, FormError>);
```

Comment:
this casting type is currently as lie, I saw that you had a FIXME here, did you have some other idea for going about fixing this?